ext/mbstring/mbstring_arginfo.h

```diff
@@ -118,6 +118,16 @@ ZEND_END_ARG_INFO()
 
 #define arginfo_mb_strtolower arginfo_mb_strtoupper
 
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_mb_trim, 0, 1, IS_STRING, 0)
+	ZEND_ARG_TYPE_INFO(0, string, IS_STRING, 0)
+	ZEND_ARG_TYPE_INFO_WITH_DEFAULT_VALUE(0, characters, IS_STRING, 0, "\" \\f\\n\\r\\t\\v\\x00                 　᠎\"")
```

Comment:
the default value from the stub, or 0x80+ chars in general, should be hex escaped

Reply:
Okay, fixed.

However, `build/gen_stub.php` is not hex escaped. This I was fix manually.

Reply:
I mean the `build/gen_stub.php` script will be good to be updated to escape 0x80+ chars, as now it looks like spaces...

Reply:
Doesn't really matter, the arginfo is not meant to be read by humans anyway. The stubs should be read instead.